apps/loan/src/components/PageLlamaMarkets/cells/SupplyYieldCell.tsx

```diff
@@ -0,0 +1,3 @@
+import { LendingVaultFromApi } from '@/entities/vaults'
+
+export const SupplyYieldCell = ({ data }: { data: LendingVaultFromApi }) => data.rates.lendApyPcent.toPrecision(4) + '%'
```

Comment:
All those number cells with percentages and dollars should probably use the `<Metric>` component in the future (it's a WIP) with all the bells and whistles turned off. Just so that there's consistency in unit font rendering and fractional rendering. But whatever, WIP, this is good enough for now.

Reply:
Good point, I'll use it once merged